modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -121,7 +116,24 @@ data:
 
 ----
 +
-See the link:https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config[Prometheus relabel_config documentation] for information about write relabel configuration options.
+.Example of forwarding metrics called `my_metric_1` and `my_metric_2` in `my_namespace` namespace
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://remote-write-endpoint.example.com"
+        writeRelabelConfigs:
+        - sourceLabels: [__name__,namespace]
+          regex: '(my_metric_1|my_metric_2);my_namespace'
```

Comment:
from
```
$ oc explain prometheus.spec.remoteWrite.writeRelabelConfigs.regex
GROUP:      monitoring.coreos.com
KIND:       Prometheus
VERSION:    v1

FIELD: regex <string>


DESCRIPTION:
    Regular expression against which the extracted value is matched.
```
I am not sure if `'(my_metric_1|my_metric_2);my_namespace'` is right or wrong

Reply:
@simonpasquier could you please confirm that it is put there correctly? :pray:

Reply:
after research, I think the expression is right, please ignore it if @simonpasquier agrees with me

Reply:
to expand the `sourceLabels` field says "concatenate the `__name__` and `namespace` labels, separated by a `;`" and the regex field says "match the concatenated string against that regex". So `(my_metric_1|my_metric_2);my_namespace` matches metrics with a name of `my_metric_1` or `my_metric_2` and a namespace label value of `my_namespace`.